android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -56,6 +57,16 @@ public String getName() {
     return REACT_CLASS;
   }
 
+  @Override
+  public void receiveCommand(final ReactViewGroup root, String commandId, ReadableArray args) {
+    // This will be called whenever a command is sent from react-native.
+    switch (commandId) {
+      case "requestAd":
+        requestAd(root);
+        break;
+    }
```

Comment:
do you think it is worthwhile to include a default here and Log.d what the command was for developer usage? I'm not familiar with the AdMob receiveCommand use case specifically, so that may be useless, but adding developer logging and including a default branch is a typical linter suggestion, so I thought I'd mention it